.gitlab-ci.yml

```diff
@@ -15,6 +15,7 @@ stages:
   - build
   - publish_artifacts
   - pre_release
+  - pre_release_image
```

Comment:
Judging by the Gitlab CI/CD, the Control Service pipelines are still run. Is this expected?

Reply:
Yes, because I have edited gitlab-ci yaml file in control service